packages/bpk-docs/src/pages/BannerAlertsPage/BannerAlertsPage.js

```diff
@@ -15,6 +15,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+/* @flow */
```

Comment:
You'll need to add the flow annotation to the index file too right?

Reply:
Yeah true, haven't done the docs yet though

Reply:
You touched this page tho :P